library/packages/src/modules/Package.rb

```diff
@@ -452,30 +455,44 @@ def LastOperationCanceled
       @last_op_canceled
     end
 
-    publish function: :by_pattern, type: "list <string> (string)"
+    # Return if system is transactional and does not support direct package
+    # install
+    def TransactionalSystem
```

Comment:
Why `TransactionalSystem` instead of `transactional_system?`? If it is just for keeping the consistency with the rest of public methods, I would call it `IsTransactionalSystem` (so it is clear it returns a boolean).

Reply:
yeap, goal is consistence. So prefix with Is sounds good to me